src/NServiceBus.Core/InitializableEndpoint.cs

```diff
@@ -43,34 +43,57 @@ public InitializableEndpoint(SettingsHolder settings, IContainer container, List
 
             var transportInfrastructure = InitializeTransport();
 
+            var isSendOnlyEndpoint = settings.Get<bool>("Endpoint.SendOnly");
```

Comment:
I think this variable is no longer needed, as we pass the settings to ReceiveComponentFactory.Cofigure as well?

Reply:
Fixed